sechub-administration/src/test/java/com/mercedesbenz/sechub/domain/administration/project/ProjectServiceTest.java

```diff
@@ -0,0 +1,164 @@
+package com.mercedesbenz.sechub.domain.administration.project;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.Set;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import com.mercedesbenz.sechub.domain.administration.user.User;
+import com.mercedesbenz.sechub.domain.administration.user.UserRepository;
+import com.mercedesbenz.sechub.sharedkernel.validation.UserInputAssertion;
+
+class ProjectServiceTest {
+
+    private ProjectService serviceToTest;
+    private UserRepository userRepository;
+
+    @BeforeEach
+    public void before() {
+        userRepository = mock(UserRepository.class);
+        UserInputAssertion userInputAssertion = mock(UserInputAssertion.class);
+
+        serviceToTest = new ProjectService(userRepository, userInputAssertion);
```

Comment:
You don't need to initialize the mocks and service class every time again.

You might as well initialize the mocks at class level once and the in the `@BeforeEach` method use `Mockito.reset(...)` (when needed)

Your service can then be initialized once as well.